code/controllers/subsystem/garbage.dm

```diff
@@ -139,7 +139,8 @@ SUBSYSTEM_DEF(garbage)
 
 	lastlevel = level
 
-	for (var/refID in queue)
+	for (var/i in 1 to length(queue))
```

Comment:
Can you leave a comment explaining why this isnt for...in?

Reply:
for you, anything